src/main/kotlin/com/sourcegraph/cody/chat/ui/ContextFilesPanel.kt

```diff
@@ -103,3 +106,12 @@ class ContextFilesPanel(
     }
   }
 }
+
+// Can pluralize "file" and "line" by adding -s
+private fun String.pluralize(count: Int): String {
+  return if (count == 1) {
+    this
+  } else {
+    "${this}s"
+  }
+}
```

Comment:
```suggestion
  private fun String.pluralize(count: Int): String = when {
    count == 1 -> this
    else -> "${this}s"
  } 
```


nit: it can be shorter while still readable